src/utils/interaction-utils.js

```diff
@@ -0,0 +1,7 @@
+class Interactions {
+  constructor() {
+    this.swiping = false;
+  }
+}
+
+export default Interactions;
```

Comment:
I was thinking of having more interactions in here, now it seems a bit of a lonely class. Will probably have to do this in a different way